tldw_Server_API/tests/AuthNZ/test_audit_chain_integration.py

```
@@ -0,0 +1,248 @@
+import pytest
+
+from tldw_Server_API.app.core.AuthNZ.audit_integrity import verify_audit_chain
+from tldw_Server_API.app.core.Audit.unified_audit_service import (
+    AuditContext,
+    AuditEvent,
+    AuditEventType,
+    UnifiedAuditService,
+)
+
+
+@pytest.fixture()
```

Ruff (0.15.6)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)